app/controllers/tags_controller.rb

```diff
@@ -181,10 +181,10 @@ def create
       return
     end
 
-    raise "Redshirt: Attempted to constantize invalid class initialize create #{type.classify}" unless Tag::TYPES.include?(type.classify)
+    raise "Redshirt: Attempted to constantize invalid class initialize create #{type.classify}" unless Tag::TYPES.include?(type.classify) || type == "Media"
```

Comment:
Instead of special-casing everywhere we use `.classify` on tag types, you could add an irregular inflection rule that "media" is both singular and plural:

https://github.com/otwcode/otwarchive/blob/fd5046c72ca2b2f6874aaaf839bd54a5410e2677/config/initializers/inflections.rb#L9